src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -108,15 +105,6 @@ private async Task AppRunningAsAdminPromptAsync()
 				UserSettingsService.ApplicationSettingsService.ShowRunningAsAdminPrompt = false;
 		}
 
-		// WINUI3
-		private ContentDialog SetContentDialogRoot(ContentDialog contentDialog)
```

Comment:
Is removing this intentional?

Reply:
Yes, this hasn't been referenced by any method.

Reply:
It's used in `AppRunningAsAdminPromptAsync()`.